traveler-swift-core/TravelerKit/Models/BoundingBox.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  BoundingBox.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-09-29.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+/**
+ A model represeting a bounding box that represents a geographic area.
+ */
+public struct BoundingBox: Equatable {
+    /// The latitude for the top left corner of the bounding box
+    public let topLeftLatitude: Double
+    /// The longitude for the top left corner of the bounding box
+    public let topLeftLongitude: Double
+    /// The latitude for the bottom right corner of the bounding box
+    public let bottomRightLatitude: Double
+    /// The longitude for the bottom right corner of the bounding box
+    public let bottomRightLongitude: Double
```

Comment:
I wonder if directional lingo is better here....I'd check with Google or other popular APIs how they model it.

i.e.

northWest, southEast, etc.


ALSO:
It makes sense to use Coordinates here no? a `Coordinate` that has lat and lng....It feels like youre repeating LAT and LNG for every point

Reply:
Agree on the coordinates model, but disagree with the directional lingo. We can't assume that the corners in a bounding box will align to a specific orientation.